LibOS/shim/include/shim_tls.h

```diff
@@ -73,6 +73,10 @@ typedef struct {
     void *                  debug_buf;
     int                     last_lock;
     struct lock_record      held_locks[NUM_LOCK_RECORD];
+    struct {
```

Comment:
Can I get a comment on the purpose of this field?

Reply:
Added.